.codesandbox/ci.json

```diff
@@ -1,4 +1,5 @@
 {
+  "node": "14",
```

Comment:
Interesting, is there a particular reason for node 14 over latest lts (16)?

Reply:
Node 14 is LTS now.

Reply:
Right, and remains in active LTS until October of this year.
16 is the next LTS https://github.com/nodejs/Release#release-schedule and will be active LTS until October of next year, giving a bit more time for updates/fixes/support.